tests/test_routes.py

```diff
@@ -46,27 +49,81 @@ def test_index(self):
         self.assertEqual(len(resp.get_json()), 6) #test number of endpoints coming back
     
     def test_create(self):
-        """ Calls create endpoint. """
-        resp = self.client.post("/promotions")
-        self.assertEqual(resp.status_code, status.HTTP_200_OK)
+        """ It should respond to a proper create with 201 status code and return the data. """
+        promo = PromoFactory()
+        data_orig = promo.serialize()
+        del data_orig['id'] # user is not supposed to send ID, they're supposed to receive it
```

Comment:
Doesn't quite make sense to me. if you delete the id, other tests can't get it by id.

Reply:
So when ben made the factory he made it with all the fields that a Promo would have in the database. Thats for backend testing

But the frontend API is made for the users to utilize. Users dont set their own ID. the ID is returned to them by the endpoint for them to keep track of or to recover from the list endpoint. 

If you're setting up tests for delete/update/read, please run a create first in your test function and then don't delete the ID AFTER the create so you can use it to test your endpoint.